package.json

```diff
@@ -55,11 +55,11 @@
   "author": "Kimmo Brunfeldt",
   "license": "MIT",
   "dependencies": {
+    "@cepharum/quoting-db": "^1.2.0",
```

Comment:
Seems like a fresh new package, would you be keen on hosting and maintaining it from the open-cli-tools org? Seems it'd fit perfectly ;)